packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesModule.java

```diff
@@ -25,7 +25,7 @@
 import expo.modules.updates.manifest.ManifestMetadata;
 
 // this unused import must stay because of versioning
-import expo.modules.updates.UpdatesConfiguration;
+
```

Comment:
delete the comment above as well?

Reply:
nice catch -- this import needs to stay actually, like the comment says. Android Studio must have automatically removed the import. I'll revert the diff